reactor-core/src/main/java/reactor/core/publisher/EmitterProcessor.java

```diff
@@ -51,7 +52,11 @@
  * @param <T> the input and output value type
  *
  * @author Stephane Maldini
- * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+ * @deprecated To be removed in 3.5. Prefer clear cut usage of {@link Sinks} through
+ * variations of <pre><code>Sinks.many().multicast().onBackpressureBuffer()</code></pre>.
+ *
+ * This processor is blocking on {@link EmitterProcessor#emitNext(T)}. This behaviour can be implemented
+ * with the {@link Sinks} API by calling {@link Sinks#many()#tryEmitNext(T)} and one of the xWhile method.
```

Comment:
`xWhile`? :)

WDYT about something like:
```suggestion
 * with the {@link Sinks} API by calling {@link Sinks#many()#tryEmitNext(T)} and retrying, e.g.:
 * <pre><code>while (sink.tryEmitNext(v).hasFailed()) LockSupport.parkNanos(10);</code></pre>
```

Reply:
yeeah I wasn't sure about how to express it, thanks for your suggestion!